src/components/Game.js

```diff
@@ -0,0 +1,43 @@
+import { useMemo, useState } from "react";
+import Board from "./Board";
+import calculateWinner from "./Winner";
+
+function Game(props) {
+  const [xIsNext, setxIsNext] = useState(true);
+  const [box, setBox] = useState(Array(9).fill(""));
+  const winner = useMemo(() => calculateWinner(box), [xIsNext]);
+  let turn;
+
+  const handleClick = (i) => {
+    if (calculateWinner(box) || box[i]) {
+      return;
+    }
+    box[i] = xIsNext ? "X" : "O";
+    setBox(box);
+    setxIsNext(!xIsNext);
+  };
+
+  if (winner) {
+    turn = winner + " is Winner!";
+  } else if (!winner && box.includes("") === false) {
+    turn = "Drawn!!!";
+  } else {
+    turn = (xIsNext ? "X" : "O") + " Play";
+  }
```

Comment:
not sure should we put this inside of `useEffect` and execute it only if `winner` has been changed or not

PS. pls correct me if im wrong 😢

Reply:
you can put this inside useMemo above and return object as output as well